project_templates/technote_aastex/{{cookiecutter.series.lower()}}-{{cookiecutter.serial_number}}/Makefile

```diff
@@ -11,7 +11,7 @@ ifneq "$(GITSTATUS)" ""
 	GITDIRTY = -dirty
 endif
 
-export TEXMFHOME = lsst-texmf/texmf
+export TEXMFHOME ?= lsst-texmf/texmf
```

Comment:
Note that by doing this the lsst-texmf that's used is no longer the one in the submodule, but rather the latest version of the lsst-texmf docker container since the container sets a TEXMFHOME environment variable.

Reply:
I did not intend to have people use a sub module for this